fabischapeli/fast-food-fast-v1
# #160244964 this deletes meal from the application

What does this PR do?

This PR adds functionality to the Meals class that can be able to delete a meal from the application
Description of Task to be completed?

The task is to make sure the meal is deleted and the message returned

 How should this be manually tested?

First fork this branch to your github and clone to your machine.
Go to google chrome, more tools, extensions and add postman extension. 
Once on postman, include the route and the method that you want to test and make sure your server is running in the terminal and click send. You will see the information <food> was deleted, 200OK else, there is error in your code or server

What are the relevant pivotal tracker stories?

#160244964
